scripts/hist_importer.py

```diff
@@ -0,0 +1,142 @@
+    parsed.add_argument('-s', '--source', dest='source', required=True,
+                        type=str, help='Source: fullpath to the sqlite data'
+                        'base file from the source browser.')
+    parsed.add_argument('-d', '--dest', dest='dest', required=True, type=str,
+                        help='Destination: The fullpath to the qutebrowser '
+                        'sqlite database')
+    return parsed.parse_args()
+
+
+def open_db(data_base):
+    """Open connection with database."""
+    try:
+        conn = sqlite3.connect(data_base)
+        return conn
+    except Exception as any_e:
+        print('Error: {}'.format(any_e))
+        raise('Error: There was some error trying to to connect with the [{}]'
+              'database. Verify if the filepath is correct or is being used.'.
+              format(data_base))
+
+
+def extract(source, query):
+    """Performs extraction of (datetime,url,title) from source."""
+    try:
+        conn = open_db(source)
+        cursor = conn.cursor()
+        cursor.execute(query)
+        history = cursor.fetchall()
+        conn.close()
+        return history
+    except Exception as any_e:
+        print('Error: {}'.format(any_e))
+        print(type(source))
+        raise('Error: There was some error trying to to connect with the [{}]'
+              'database. Verify if the filepath is correct or is being used.'.
+              format(str(source)))
+
+
+def clean(history):
+    """Receives a list of records:(datetime,url,title). And clean all records
+    in place, that has a NULL/None datetime attribute. Otherwise Qutebrowser
+    will throw errors. Also, will add a 4rth attribute of '0' for the redirect
+    field in history.sqlite in qutebrowser."""
+    nulls = [record for record in history if record[0] is None]
+    for null_datetime in nulls:
+        history.remove(null_datetime)
+    history = [list(record) for record in history]
+    for record in history:
+        record.append('0')
+    return history
+
+
+def insert_qb(history, dest):
+    """Given a list of records in history and a dest db, insert all records in
+    the dest db."""
+    conn = open_db(dest)
+    cursor = conn.cursor()
+    cursor.executemany(
+        'INSERT INTO History (url,title,atime,redirect) VALUES (?,?,?,?)',
+        history
+    )
+    cursor.execute('DROP TABLE CompletionHistory')
+    conn.commit()
+    conn.close()
+
+
+def main():
+    """Main control flux of the script."""
+    args = parser()
+    browser = args.browser.lower()
+    source, dest = args.source, args.dest
+    query = {
+        'firefox': 'select url,title,last_visit_date/1000000 as date '
+                   'from moz_places',
+        'chrome': 'select url,title,last_visit_time/10000000 as date '
+                  'from urls',
+        'safari': None
+    }
+    if browser not in query:
+        sys.exit('Sorry, the selected browser: "{}" is not supported.'.format(
+            browser))
```

Comment:
After passing `choices` to argparse above, this check is probably not needed anymore.